### ISSUE TITLE
set execute bit only on executables

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#40]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/40) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

currently all files in the development repostiory seems to have execute bit 

on a linux System we should may be remove first all ritghts and set them afterwards to

```
#!shell


cd mss
chmod -R a-rwx
chmod -R a+rwX
chmod a+x mslib/msui/mss_pyui.py
chmod a+x mslib/mswms/mss_wms_cl.py
chmod a+x mslib/mswms/mss_wms_wsgi.standalone_paste.py
```

The last one will be replaced by e.g. mswms.py

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

only directories and a few files should have execute bit set, fixes #40

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss (pull request #61)

only directories and a few files should have execute bit set, fixes #40
